hack/generate-proto.sh

```diff
@@ -11,7 +11,7 @@ ensure_vendor
 
 make_fake_paths
 # This overrides the existing trap
-trap 'cd "${FAKE_GOPATH}" && go clean -modcache && cd /tmp && rm -rf "${FAKE_GOPATH}"' EXIT
+trap 'cd "${FAKE_GOPATH}" && go clean -modcache && rm -rf "${FAKE_GOPATH}"' EXIT
```

Comment:
@tmenjo - Can not think of a reason why it fails in your env, but your approach is harmless.